styles.css

```diff
@@ -561,37 +561,16 @@ section:first-child {
 .project {
   width: 100%;
   border: 3px solid;
-  display: flex;
-  flex-direction: column;
   margin-bottom: 2em;
+  flex: 1 1 100%;
 }
 
-@media screen and (min-width: 870px) {
+@media screen and (min-width: 930px) {
   .project {
-    width: calc(50% - 1em);
+    flex: 0 1 438px;
```

Comment:
At some middle widths, when there are only two columns, there's now a very wide gap between the columns (as wide as 437px). Do you think it'd be worth having an additional breakpoint for this case? Glad to implement if so.

Reply:
Yeah, if you'd like to restore that functionality and have an idea on how to do that, go for it!

Reply:
```suggestion
    flex: 0 1 calc(50% - 1em);
  }
}

@media screen and (min-width: 1361px) {
  .project {
    flex: 0 1 438px;
```